### DISCUSSION TITLE
GPS data from phone instead of ATM

### DISCUSSION THREAD

**Discussion by IrakliyBerison** [BimmerGestalt/AAIdrive#732] [Ideas]
Hi, great app. Great opportunities.

Sadly there is no app for iOS. But i have android to 🙂
So, is it possible to use phone like GPS receiver and send data to NBT Evo?

Some additional info.
I’ve upgraded my NBT to NBT Evo, but don’t upgraded TCB to ATM(hope you know this  letters meaning)
And now, when i’m entering to navigation, it’s cant find location or navigate..

**Comment by hufman** [comment]
Thanks for the compliments!
Unfortunately I don't know any method to send GPS data to the car without installing extra hardware.
